docs/tour/_category_.json

```diff
@@ -1,8 +1,8 @@
 {
-    "label": "Getting Started",
-    "position": 2,
-    "link": {
-        "type": "generated-index",
-        "description": "Get Started using wasmCloud"
-    }
-}
\ No newline at end of file
+  "label": "Getting Started",
+  "position": 2,
+  "link": {
+    "type": "generated-index",
+    "description": "Now that you've installed wash, it's time to get started with wasmCloud and dive into a better way to write applications."
```

Comment:
nitpick: can we make this more active voice, maybe "Now that you've installed wash, dive into a better way to write applications with wasmCloud"